### PR TITLE
solve footer overlaping

### LINKED ISSUE #565: footer is overlapping guide.hbs.
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behavior:**

**Expected behavior:**

No overlapping.
**Steps to reproduce:**
go to http://badgeyay.com/#/guide and scroll down to the bottom. You will see overlapping of footer on guide.hbs template.

**Do you Want to work on it ?**
No, a beginner can take up this issue.

### PR DESCRIPTION
**Pull request by avsingh999**
Fixes #565

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- update frontend/app/templates/guide.hbs
- update frontend/public/css/style.css

### CODE REVIEW

**Comment by pushkalkatara** [comment]
Please squash your commits @avsingh999.

**Comment by avsingh999** [comment]
@ParthS007 what's wrong in codecov report please tell me.

File: frontend/app/templates/guide.hbs [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
+                <pre class="alert-info">Format for Manual data is same as CSV's data</pre>
+                <p>Five comma (',') seperated values on each line is the correct format</p>
+                <p>See Image below</p>
+                <img src="/images/guide_manual.png" class="guide_img" alt="Manual Data Image">
+            </div>
         </div>
     </div>
 </div>
-
-<script src="{{rootURL}}js/script.js"></script>
```
Comment (rupav) [inline frontend/app/templates/guide.hbs]: This must not be removed. Please restore it.

File: frontend/public/css/style.css [hunk 2]
```diff
@@ -6,6 +6,12 @@
     top:3px;
 }
 
+.col-md-12 {
+    margin-top: 10px;
+    margin-bottom: 60px;
+    padding-bottom: 20px;
```
Comment (rupav) [inline frontend/public/css/style.css]: set margin-bottom to 30px instead, otherwise 60px is creating too much space.

**Comment by rupav** [comment]
@avsingh999 there is nothing wrong with codecoverage report. You should read about it online how it works.

File: frontend/app/templates/guide.hbs (hunk 1 above)
Comment (avsingh999) [reply frontend/app/templates/guide.hbs]: ok

File: frontend/public/css/style.css [hunk 3]
```diff
@@ -6,6 +6,12 @@
     top:3px;
 }
 
+.col-md-12 {
+    margin-top: 10px;
+    margin-bottom: 30px;
+    padding-bottom: 20px;
```
Comment (abishekvashok) [inline frontend/public/css/style.css]: Can you make this footer specific as we might have problems on other pages as this is a site wide style.css

Comment (avsingh999) [reply frontend/public/css/style.css]: @abishekvashok i am not getting your point.will you explain more

Comment (abishekvashok) [reply frontend/public/css/style.css]: This change is applicable to all elements of the col-md-12 class. That's unwanted. Can you narrow the changes to footer only?

**Comment by abishekvashok** [comment]
@avsingh99 any updates?

**Pull request closed without merging**